backend/tests/apps/owasp/graphql/nodes/project_health_metrics_test.py

```
@@ -0,0 +1,69 @@
+import pytest
+
+from apps.owasp.graphql.nodes.project_health_metrics import ProjectHealthMetricsNode
+
+
+class TestProjectHealthMetricsNode:
+    def test_project_health_metrics_node_inheritance(self):
+        assert hasattr(ProjectHealthMetricsNode, "__strawberry_definition__")
+
+    def test_meta_configuration(self):
+        field_names = {
+            field.name for field in ProjectHealthMetricsNode.__strawberry_definition__.fields
```

Pylint (3.3.7)

[error] Class 'ProjectHealthMetricsNode' has no '__strawberry_definition__' member

(E1101)